src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/Result/Cells/TechnicalValidationFailedRulesTableViewCell.swift

```diff
@@ -27,10 +27,33 @@ class TechnicalValidationFailedRulesTableViewCell: UITableViewCell, ReuseIdentif
 
 		updateBorderColor()
 	}
+	
+	override func prepareForReuse() {
+		super.prepareForReuse()
+		clearErrorViews()
+	}
 
+	// MARK: - Internal
+	
+	func customize(signatureInvalid: Bool, expirationDate: Date?) {
+		
+		clearErrorViews()
+				
+		if signatureInvalid {
+			addErrorViewSeparator()
+			errorViewsStackView.addArrangedSubview(TechnicalValidationFailedRulesTableViewCellErrorView(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.certificateNotValid, expirationDate: nil))
+		}
+		
+		if let expirationDate = expirationDate {
+			addErrorViewSeparator()
```

Comment:
```suggestion
			if !errorViewsStackView.arrangedSubviews.isEmpty {
				addErrorViewSeparator()
			}
```

Only makes sense if there is an error shown above